README.md

```diff
@@ -15,37 +15,28 @@ To get access to the GAE, [see the documentation on Slab](https://uclh.slab.com/
 
 ## Installation
 
-Install the PIXL CLI by running
+Install the PIXL Python modules by running the following commands from the top-level `PIXL/` directory:
 
 ```shell
-pip install -e cli/
+python -m pip install pixl_core/
+python -m pip install pytest-pixl/
+python -m pip install -e cli/
+python -m pip install pixl_imaging/
+python -m pip install pixl_dcmd/
+python -m pip install pixl_export/
+python -m pip install hasher/
```

Comment:
So for a deployment you only need to install `core` and `cli`, the others are all used within docker. Then the development instructions can let you know how to install the dev and test

both of these need a `-e`, what's fun is that if you ever install without -e, then it never is installed in editable mode and you need to kill your environment and start again.

Reply:
fun! Just wondering why core also needs to be editable (as far as I could see, only the CLI uses `PIXL_ROOT`)? I've updated the installation instructions now

Reply:
ah, that's more of avoiding issues if there's updates to the module that are still in the same version. We've had a lot of fun with that, expecting it to update when it doesn't

Reply:
oh I see, that makes sense, thanks